docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
       KC_DB_USERNAME: sa
       KC_DB_PASSWORD: D3velopmentP0
       KEYCLOAK_ADMIN: admin
-      KEYCLOAK_ADMIN_PASSWORD: D3velopmentP0
+      KEYCLOAK_ADMIN_PASSWORD: g
```

Comment:
we just want to make this easier for us to type the password?

Reply:
oh! I think I accidentally make this change